### PR TITLE
Add brief contribution guide

### LINKED ISSUE #16: Contributor's Guide
We need to formulate a Contributor's Guide document that describes the workflow used for adding contributions.
The [FPC Git flows](https://docs.google.com/document/d/1sR7YV3pSYN3NEFiW-2fUqtpsJeJrpC0EWUVtEm0Blcg/edit) can serve as a great starting point.

### PR DESCRIPTION
**Pull request by mbrandenburger**
Provide a first description of the workflow used for adding
contributions to MirBFT. This is intended as an initial version,
based on the experience gathered with Fabric Private Chaincode,
and to be continuously developed further over the time.

Signed-off-by: Marcus Brandenburger <bur@zurich.ibm.com>

This fixes #16

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+## Contributing
+
+We appreciate your contributions to MirBFT.
+Any help is welcome and there is still much to do. 
+
+We adopt Fabric's developer guidelines where possible. Please visit Fabric's [contributors guide](http://hyperledger-fabric.readthedocs.io/en/latest/CONTRIBUTING.html) in the
+docs to learn more about it.
+
+Note that we follow the Hyperledger [Charter](https://www.hyperledger.org/about/charter), particularly,
+all new inbound code contributions to Fabric Private Chaincode shall be made under the Apache License, Version 2.0.
```
Comment (matejpavlovic) [inline CONTRIBUTING.md:14]: Fabric Private Chaincode -> MirBFT

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+### A brief contribution guide
+
+#### Everything starts with an issue
+
+Contributions to MirBFT usually start by creating a [new issue](https://github.com/hyperledger-labs/mirbft/issues) to report a bug or suggest a new feature.
+If you are reporting a bug, please try to provide enough information to allow someone else to reproduce the bug.
+For instance, tell us which version of the code (i.e., which commit or release) you are using, etc. 
+If you are proposing a new feature or enhancement, please describe the problem you are trying to solve with enough details.
+Explain the solution you have in mind. Try to formulate a clear and concise description of what you want to happen. Have you considered any alternatives?
+Next, it is time to trigger a discussion on your new issue and to make this an interactive process to reach agreement on a solution. Feel free to post your issue also on #mirbft Rocket Chat and asked for feedback.
```
Comment (matejpavlovic) [inline CONTRIBUTING.md:26]: ask for feedback

**Review by matejpavlovic — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
```
Comment (matejpavlovic) [inline CONTRIBUTING.md:4]: Should we keep this as CC or change it to Apache 2.0 as well as the code?

Comment (mbrandenburger) [reply CONTRIBUTING.md:4]: Documentation goes under CC normally, https://wiki.hyperledger.org/display/TSC/Copyright+and+License+Policy

**Review by matejpavlovic — APPROVED** [review]
Looks good to me!

**Pull request merged**